playbooks/provisioning/openstack/provision-openstack.yml

```diff
@@ -39,14 +39,18 @@
   - meta: refresh_inventory
 
 - hosts: cluster_hosts
+  name: Wait for the the nodes to come up
+  become: False
+  gather_facts: False
```

Comment:
@bogdando we can't gather facts before the nodes are accessible (and `gather_facts` happens before `tasks`  so before `wait_for_connection` happens).

If we need to gather facts in this particular section, we can add this task after wait_for_connection:

      - name: Gather facts for the created nodes
        setup:

But why would we need to? Nothing uses them in this particular play, and they're gathered right after this one.